src/meta/src/hummock/manager/compaction.rs

```diff
@@ -265,6 +266,67 @@ impl HummockManager {
             } => {}
         }
     }
+
+    pub(crate) async fn calculate_vnode_partition(
+        &self,
+        compact_task: &mut CompactTask,
+        compaction_config: &CompactionConfig,
+    ) {
+        if compaction_config.split_weight_by_vnode > 0 {
+            for table_id in &compact_task.existing_table_ids {
+                compact_task
+                    .table_vnode_partition
+                    .insert(*table_id, compact_task.split_weight_by_vnode);
+            }
+        } else {
```

Comment:
related to https://github.com/risingwavelabs/risingwave/pull/16495

we prohibit splitting sst at high level, I think this pr also needs to conform to this design. What do you think?

Reply:
Ok, I will skip it in high level, it is not a problem.

Reply:
I checked the code and I found that it has cleared vnode partition